webpack/assets/javascripts/react_app/components/common/table/formatters/actionsCellFormatter.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { translate as __ } from '../../../../common/I18n';
+
+export const deleteActionCellFormatter = controllerPluralize => (
+  _,
+  { rowData: { can_delete: canDelete, name, id } },
+) =>
+  // eslint-disable-next-line camelcase
+  (canDelete ? (
+    <span className="btn btn-sm btn-default">
```

Comment:
use a Button component?

Reply:
I am not sure about this now. It appears that using `Button` instead of `span` changes the look of the action button.